### PR TITLE
Bugfix/improve quality

### PR DESCRIPTION
**Pull request by jbern0rd**

### CODE REVIEW

File: src/test/java/com/iexec/blockchain/version/VersionServiceTests.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    public void preflight() {
+        MockitoAnnotations.openMocks(this);
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings={"x.y.z", "x.y.z-rc"})
+    void testNonSnapshotVersion(String version) {
+        Mockito.when(buildProperties.getVersion()).thenReturn(version);
+        Assertions.assertEquals(version, versionService.getVersion());
+        Assertions.assertFalse(versionService.isSnapshot());
```
Comment (jeremyjams) [inline src/test/java/com/iexec/blockchain/version/VersionServiceTests.java:48]: You can remove `isSnapshot` method instead

Comment (jbern0rd) [reply src/test/java/com/iexec/blockchain/version/VersionServiceTests.java:48]: If we remove `isSnapshot`, why not simply removing `VersionService` and directly use `BuildProperties` when necessary ?
Do we want to go in this direction ?

**Review by jeremyjams — APPROVED** [review]
Thank you for the update :+1:

**Review by mcornaton — APPROVED** [review]
Unused import [there](https://github.com/iExecBlockchainComputing/iexec-blockchain-adapter-api/blob/c350c96aff08d3bfaebbd322450b2ff3ca657c3d/src/main/java/com/iexec/blockchain/version/VersionService.java#L19). Otherwise, it's alright for me. Thanks!

**Comment by jbern0rd** [comment]
> Unused import [there](https://github.com/iExecBlockchainComputing/iexec-blockchain-adapter-api/blob/c350c96aff08d3bfaebbd322450b2ff3ca657c3d/src/main/java/com/iexec/blockchain/version/VersionService.java#L19). Otherwise, it's alright for me. Thanks!

Fixed in f083723, thanks

**Pull request merged**